cmd/experimental/kueue-populator/charts/kueue-populator/values.yaml

```diff
@@ -4,7 +4,7 @@ kueuePopulator:
     # -- Populator's image repository
     repository: us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue-populator
```

Comment:
@mbobrovskyi we should update it for the release to the registry.k8s.io, right?

Reply:
Yes, it should be updated automatically. It doesn't work?

Reply:
Also as you can see we don't touch main charts https://github.com/mimowo/kueue/blob/0276ad4281ac149e295439e8473d8f3dba6a7eb9/charts/kueue/values.yaml#L29.